src/backend/api/Fusion.Resources.Domain/Queries/GetResourceAllocationRequestsForAnalyticsAsync.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Fusion.Resources.Database;
+using MediatR;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Fusion.AspNetCore.OData;
+using Fusion.Integration.Diagnostics;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+
+namespace Fusion.Resources.Domain.Queries;
+
+public class GetResourceAllocationRequestsForAnalyticsAsync : IRequest<IAsyncEnumerable<QueryResourceAllocationRequest>>
```

Comment:
Rename async suffix to stream?
```suggestion
public class GetResourceAllocationRequestsForAnalyticsStream : IRequest<IAsyncEnumerable<QueryResourceAllocationRequest>>
```